board/provision.h

```diff
@@ -5,7 +5,7 @@
 // SHA1 checksum = 0x1C - 0x20
 
 void get_provision_chunk(uint8_t *resp) {
-  (void)memcpy(resp, (void *)0x1fff79e0, PROVISION_CHUNK_LEN);
+  (void)memcpy(resp, (uint8_t *)0x1fff79e0, PROVISION_CHUNK_LEN);
```

Comment:
this was failing 11.6: A cast shall not be performed between pointer to void and an arithmetic type